src/renderers/shared/server/ReactPartialRenderer.js

```diff
@@ -484,6 +484,19 @@ class ReactDOMServerRenderer {
       if (child === null || child === false) {
         return '';
       } else {
+        if (Array.isArray(child)) {
+          var frame = {
+            children: child,
+            childIndex: 0,
+            context: context,
+            footer: '',
+          };
+          if (__DEV__) {
+            frame.debugElementStack = [];
```

Comment:
like this?
`itRenders('a array type children as a child', async render => {
        ReactDOMServer.renderToString(
          [<span key={1}>text1</span>, <p key={2}>text2</p>],
        ); 
`